run.py

```
@@ -0,0 +1,67 @@
+import logging
+import signal
+import sys
+import time
+from optparse import SUPPRESS_HELP
```

Ruff

`optparse.SUPPRESS_HELP` imported but unused

Remove unused import: `optparse.SUPPRESS_HELP`

(F401)